core/templates/components/button-directives/social-buttons.component.spec.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { UrlInterpolationService } from 'domain/utilities/url-interpolation.service';
+import { MockTranslatePipe } from 'tests/unit-test-utils';
+import { SocialButtonsComponent } from './social-buttons.component';
+
+/**
+ * @fileoverview Unit tests for SocialButtonsComponent
+ */
+
+describe('SocialButtonsComponent', () => {
+  let component: SocialButtonsComponent;
+  let fixture: ComponentFixture<SocialButtonsComponent>;
+  let urlInterpolationService: UrlInterpolationService;
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        SocialButtonsComponent,
+        MockTranslatePipe
+      ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(SocialButtonsComponent);
+    component = fixture.componentInstance;
+    urlInterpolationService = TestBed.inject(UrlInterpolationService);
+  });
+
+  it('should successfully instantiate the component from beforeEach block',
+    () => {
+      expect(component).toBeDefined();
+    });
```

Comment:
It is just a check to see if the component has been created successfully or not. This is the first test I right to check If the setup was correct.

Removed file (because of the comment below)